specification/confluent/resource-manager/Microsoft.Confluent/preview/2020-03-01-preview/confluent.json

```diff
@@ -33,6 +33,78 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Confluent/agreements": {
+        "get": {
+          "tags": [
+            "Agreements"
+          ],
+          "summary": "List Confluent marketplace agreements in the subscription.",
+          "operationId": "MarketplaceAgreements_List",
+          "produces": [
+            "application/json"
+           ],
+           "parameters": [
+              {
+                "$ref": "#/parameters/SubscriptionIdParameter"
+              }
+           ],
+           "responses": {
+              "200": {
+                "description": "Success",
+                "schema": {
+                  "$ref": "#/definitions/AgreementTermsListResponse"
+                }
+            },
+            "default": {
+               "description": "Default error response.",
+               "schema": {
+                 "$ref": "#/definitions/ResourceProviderDefaultErrorResponse"
+               }
+            }
+          },
+          "x-ms-pageable": {
+              "nextLinkName": "nextLink"
+          },
+          "x-ms-examples": {
+            "MarketplaceAgreements_List": {
+              "$ref": "./examples/MarketplaceAgreements_List.json"
+            }
+          }
+        }
+     },
+     "/subscriptions/{subscriptionId}/providers/Microsoft.Confluent/agreements/all/acceptTerms": {
```

Comment:
This API is problematic from ARM's perspective. See the following comment on a different PR: https://github.com/Azure/azure-rest-api-specs-pr/pull/2309#discussion_r514506693. I'm starting a mail thread to discuss this...